src/packer.pkr.hcl

```diff
@@ -149,13 +149,189 @@ build {
   }
 
   provisioner "powershell" {
-    # Wait 90 seconds before executing the check-defender.ps1 powershell script.
-    # This gives a generous grace period between restarting Windows and running the second provisioner.
-    pause_before = "90s"
-    scripts      = ["src/powershell/check-defender.ps1"]
+    # Wait 30 seconds before executing the next provisioner.
+    # This gives a grace period between restarting Windows and running the next provisioner.
+    pause_before = "30s"
+    scripts = [
+      "src/powershell/check-defender.ps1",
+      "src/powershell/enable-rdp.ps1",
+      "src/powershell/install-chocolatey.ps1"
+    ]
   }
 
   provisioner "powershell" {
-    scripts = ["src/powershell/enable-rdp.ps1"]
+    # Create "packages" directory before uploading them in the next provisioner.
+    inline = ["mkdir C:\\packages"]
```

Comment:
Would you create an issue to turn this into a build variable (it can use `C:/packages` as a default)? The `install-category.ps1` script should be updated to use a new environment variable that would be the value of the build variable to locate the package lists. I don't want to slow down this immediate effort but I abhor hard-coded values that are not OS mandated.

Reply:
@mcdonnnj Can i please get more context on this so i can create an issue for you? Thanks!

Reply:
Currently the location to install packages is hard-coded. I would like the package installation location to instead be a build argument with a default value matching the current hard-coded value in the Packer configuration. This is both better design and easier maintainability in case the install location needs to change for any reason.

Reply:
@mcdonnnj resolved with [cb9ed08](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/cb9ed08d2cae41fb075efb1c99d257004ac5c117)